### ISSUE TITLE
Navigation should belong within a `<header>` landmark outside the `<main>`

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org#363]
### Description of issue
Global navigations that remain consistent page by page should live within the `<header>` landmark element outside of the `<main>`. This helps both assistive tech users and SEO.

### Steps to reproduce
1. Use this landmarks extension to view the landmarks of a page https://chrome.google.com/webstore/detail/landmark-navigation-via-k/ddpokpbjopmeeiiolheejjpkonlkklgp?hl=en-US
2. Confirm the global navigation is nested under the main

### Screenshot

https://user-images.githubusercontent.com/14154792/129051681-7550865a-6937-407e-8567-1d26d3828f85.mov

```
Path:      /
Browser:   Chrome 92.0.4515.131 on Mac OS 10.15.7
Viewport:  887 x 952 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-359--dcdesignweek.netlify.app%2F&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=92.0&resolution=887x952&speed=1&start=true&ref=netlify-source)

Submitted from [Netlify Deploy Preview #359](https://deploy-preview-359--dcdesignweek.netlify.app/)

**Comment by marcusrelacion** [comment]
This has been updated @joshkimux:

<img width="1449" alt="CleanShot 2021-08-11 at 18 03 16@2x" src="https://user-images.githubusercontent.com/6185899/129109446-72828329-51f7-4262-8e1b-c512e2efacf5.png">

<img width="397" alt="CleanShot 2021-08-11 at 18 02 15@2x" src="https://user-images.githubusercontent.com/6185899/129109358-2945dc88-a5d2-4b24-adcc-9cab6840e202.png">
